src/DssSpell.sol

```diff
@@ -55,107 +59,160 @@ contract DssSpellAction is DssAction {
+
+    // ---------- Increase RWA014-A (Coinbase Custody) Debt Ceiling ----------
+    address internal immutable MIP21_LIQUIDATION_ORACLE = DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
+
+    // ---------- Andromeda Legal Expenses ----------
+    address internal constant BLOCKTOWER_WALLET_2 = 0xc4dB894A11B1eACE4CDb794d0753A3cB7A633767;
 
     // ---------- Trigger Spark Proxy Spell ----------
-    address internal constant SPARK_SPELL            = 0xDa69603384Ef825E52FD5B8bEF656ff62Fe19703;
-
-    // ---------- Whistleblower Bounty ----------
-    address internal constant VENICE_TREE            = 0xCDDd2A697d472d1e8a0B1B188646c756d097b058;
-
-    // ---------- Launch Project Funds ----------
-    address internal constant LAUNCH_PROJECT_FUNDING = 0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F;
-
-    // ---------- Delegates ----------
-    address internal constant DEFENSOR               = 0x9542b441d65B6BF4dDdd3d4D2a66D8dCB9EE07a9;
-    address internal constant TRUENAME               = 0x612F7924c367575a0Edf21333D96b15F1B345A5d;
-    address internal constant BONAPUBLICA            = 0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3;
-    address internal constant CLOAKY                 = 0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818;
-    address internal constant NAVIGATOR              = 0x11406a9CC2e37425F15f920F494A51133ac93072;
-    address internal constant VIGILANT               = 0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61;
-    address internal constant UPMAKER                = 0xbB819DF169670DC71A16F58F55956FE642cc6BcD;
-    address internal constant PBG                    = 0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2;
-    address internal constant PALC                   = 0x78Deac4F87BD8007b9cb56B8d53889ed5374e83A;
-    address internal constant BLUE                   = 0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf;
-    address internal constant JAG                    = 0x58D1ec57E4294E4fe650D1CB12b96AE34349556f;
+    // Spark Proxy: https://github.com/marsfoundation/sparklend/blob/d42587ba36523dcff24a4c827dc29ab71cd0808b/script/output/1/primary-sce-latest.json#L2
+    address internal constant SPARK_PROXY = 0x3300f198988e4C9C63F75dF86De36421f06af8c4;
+    address internal constant SPARK_SPELL = 0x68a075249fA77173b8d1B92750c9920423997e2B;
+
+    // ---------- Payment Streams ----------
+    address internal immutable MCD_VEST_DAI          = DssExecLib.getChangelogAddress("MCD_VEST_DAI");
+    address internal immutable MCD_VEST_MKR_TREASURY = DssExecLib.getChangelogAddress("MCD_VEST_MKR_TREASURY");
+    GemAbstract internal immutable MKR               = GemAbstract(DssExecLib.mkr());
+    address internal constant ECOSYSTEM_FACILITATOR  = 0xFCa6e196c2ad557E64D9397e283C2AFe57344b75;
+
+    // ---------- Timestamps ----------
+    // 2023-12-01 00:00:00
+    uint256 internal constant DEC_01_2023 = 1701385200;
+    // 2024-11-30 23:59:59
+    uint256 internal constant NOV_30_2024 = 1733007599;
```

Comment:
Not big deal, but in general the variable name indicated the day at 0:00 UTC time.
Now if we look at the comments, both variables use different criteria between each other which can be confusing but still I guess it is a subjective decision.
However both dates are actually shifted by one hour in UTC according to the comment.
Again not a big deal, and maybe it was discussed internally, but just wanted to leave a comment for future awareness.

Reply:
Thanks for reviewing the spell and providing feedback! 

Technically, the governance instruction does not contain timezone information and checklists doesn't mention UTC as a requirement, so it can't be regarded as a mistake/error. But I agree that we can better standardise this to avoid inconsistencies, so I created an [improvement proposal](https://github.com/makerdao/pe-checklists/pull/16). Feel free to comment on it directly.

Reply:
Yeah I'd guess UTC is the obvious default as any other timezone would be just a random choice for a world wide decentralized project. But being explicit in the guideline is just better.